static/src/javascripts/projects/common/modules/identity/manage-account.js

```diff
@@ -126,6 +165,27 @@ const confirmUnsubscriptionFromAll = buttonEl => {
     });
 };
 
+const bindHtmlPreferenceChange = (buttonEl: HTMLButtonElement): void => {
+    bean.on(buttonEl, 'click', () =>
```

Comment:
`buttonEl.addEventListener()` ?

Reply:
i kinda wanna refactor out bean all at once if that's okay. I take it the worst part of having it around is the import and not the overhead of each individual call?